addon/services/redux.js

```diff
@@ -0,0 +1,50 @@
+import Ember from 'ember';
+import redux from 'redux';
+import reducers from '../reducers/index';
```

Comment:
Since these imports are within the addon itself, should we actually exclude them here, and just rely on them being passed into the extended version of the service? I don't think they add any value here since we want to include the `reducers`, `enhancers`, and `middlewareConfig` from the host app anyway.